tooling/config-eslint/utils/overrides.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    overrides: [
+        {
+            files: ['*.cjs', '*.js', '*.mjs'],
+            parser: '@typescript-eslint/parser',
+        },
+        {
+            extends: ['plugin:@typescript-eslint/disable-type-checked'],
+            files: ['.eslintrc.js'],
```

Comment:
```plaintext
  0:0  error  Parsing error: /Users/josh/repos/typehero/.eslintrc.js was not found by the project service. Consider either including it in the tsconfig.json or including it in allowDefaultProject
```

Because there's no root-level `tsconfig.json`, the root-level `.eslintrc.js` isn't matched to any TSConfig project settings.